src/Microsoft.Composition.Analyzers/Core/DoNotMixAttributesFromDifferentVersionsOfMEF.cs

```diff
@@ -1,15 +1,19 @@
 // Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using System.Collections.Generic;
 using System.Collections.Immutable;
-using Microsoft.CodeAnalysis.Diagnostics;
+using System.Linq;
+using Analyzer.Utilities;
 using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
 
 namespace Microsoft.Composition.Analyzers
 {
     /// <summary>
     /// RS0006: Do not mix attributes from different versions of MEF
     /// </summary>
-    public abstract class DoNotMixAttributesFromDifferentVersionsOfMEFAnalyzer : DiagnosticAnalyzer
+    [DiagnosticAnalyzer(LanguageNames.CSharp, LanguageNames.VisualBasic)]
+    public sealed class DoNotMixAttributesFromDifferentVersionsOfMEFAnalyzer : DiagnosticAnalyzer
     {
         internal const string RuleId = "RS0006";
```

Comment:
Can you give them some CA ids? Mike has a list of all used FxCop ids that we can use to get the next free one.